### PR TITLE
Formatting and typo issues in qiskit.transpiler.Target

### PR DESCRIPTION
**Pull request by 1ucian0**
### Summary

Some small typo issues in `qiskit.transpiler.Target`

### CODE REVIEW

File: qiskit/transpiler/target.py [hunk 1]
```diff
@@ -1034,14 +1034,14 @@ def build_coupling_map(self, two_q_gate=None, filter_idle_qubits=False):
...
-            IndexError: If an Instruction not in the Target is passed in for
+            IndexError: If an Instruction not in the ``Target`` is passed in for
                 ``two_q_gate``.
         """
         if self.qargs is None:
             return None
         if None not in self.qargs and any(len(x) > 2 for x in self.qargs):
             logger.warning(
-                "This Target object contains multiqubit gates that "
+                "This Target object contains multi-qubit gates that "
```
Comment (jakelishman) [inline qiskit/transpiler/target.py]: "multiqubit" is fine (especially in US English), it's just a constructed word.

File: qiskit/transpiler/target.py [hunk 2]
```diff
@@ -817,7 +817,7 @@ def check_obj_params(parameters, obj):
             if parameters is not None:
                 obj = self._gate_name_map[operation_name]
                 if inspect.isclass(obj):
-                    # The parameters argument was set and the operation_name specified is
+                    # The parameters' argument was set and the operation_name specified is
```
Comment (jakelishman) [inline qiskit/transpiler/target.py]: The apostrophe here isn't correct - it should either be as it was before, or "The 'parameters' argument" (though also, this is just a code comment).

File: qiskit/transpiler/target.py (hunk 1 above)
Comment (ElePT) [reply qiskit/transpiler/target.py]: Also a small note here, should this change stay, the log assert in the target unit test would have to be modified to add the hyphen (this is why CI currently fails): https://github.com/Qiskit/qiskit/blob/main/test/python/transpiler/test_target.py#L558

File: qiskit/transpiler/target.py (hunk 2 above)
Comment (1ucian0) [reply qiskit/transpiler/target.py]: Undone in 8c4650414d3dac6c960948e648458f0d495dfd4a. Thanks!

**Review by jakelishman — APPROVED** [review]

**Pull request merged**